innowiseTasks

```diff
@@ -0,0 +1,24 @@
+
+USE InnowiseSQLTASK;
+
```

Comment:
First of all we need to create a tables and fill data. You should be able to do it in first order.